config/user_preferences_extra_conf.xml.sample

```diff
@@ -0,0 +1,53 @@
+preferences:
+    # the key you can refer to
+    apollo_url_01:
+        # description that is displayed to the user
+        description: The URL to your personal Apollo instance
+        inputs:
+            # the content can be accessed by apollo_url_01.apollo_url
+            - name: apollo_url
+              label: Apollo URL
+              # type of input field that will be displayed to the user
+              # can be string or password
+              type: text
+              # by defaul all inputs are required
+              required: True
+            - name: appollo_text
+              label: Apollo Text
+              # type of input field that will be displayed to the user
+              # can be string or password
+              type: text
+              # by defaul all inputs are required
+              required: True
+
+    openstack_account:
+        description: Your own Open Stack account
+        inputs:
+            - name: username
+              label: Username
+              type: text
+              required: False
+            - name: password
+              label: Password
+              type: password
+              required: False
+            - name: url
+              label: Open Stack URL
+              type: text
+              required: True
+            - name: url_2
```

Comment:
why is there a `url_2`? Shouldn't it just be a second instance of an `openstack_account` type preference?